### ISSUE TITLE
Error: EROFS: read-only file system

### ISSUE THREAD

**Issue by Guillaume-Levrier** [Guillaume-Levrier/PANDORAE#6]
Some macOS devices have the following error on first app launch:

```
A JavaScript error occurred in the main process

Uncaught Exception:
Error: EROFS: read-only file system, open '/private/var/folders/XXXXXX/T/AppTranslocation/XXXXX/d/PANDORAE.app/Contents/Resources/app/json/themes.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:166:20)
    at Object.fs.writeFileSync (fs.js:1303:33)
    at fs.readFile (/private/var/folders/XXXX/T/AppTranslocation/XXXXXX/d/PANDORAE.app/Contents/Resources/app/main.js:56:10)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
```

themes.json apparently can't be written in the folder, maybe because folder creation isn't over when it tries to do so. Restarting PANDORÆ once (or twice) seems to fix the problem.

**Comment by Guillaume-Levrier** [comment]
Putting the App in the user's "Applications" folder apparently solves the problem in some cases.

**Comment by Guillaume-Levrier** [comment]
Issue should be solved by commit [cdfb7f337fafe45c9aa73f25271269dbd48cfac4](https://github.com/Guillaume-Levrier/PANDORAE/commit/cdfb7f337fafe45c9aa73f25271269dbd48cfac4)
